packages/vaex-meta/setup.py

```diff
@@ -21,7 +21,8 @@
       'vaex-server==0.2',
       'vaex-hdf5==0.4',
       'vaex-astro==0.4',
-      'vaex-arrow==0.3'
+      'vaex-arrow==0.3',
+      'vaex-ml==0.4'
```

Comment:
I don't think vaex-ml should be installed by default when you install vaex, like vaex-ui, what do you think?

Reply:
Especially since vaex-ml now still depends on numba.

Reply:
Why not?  I thought it is good to come as a default. I am afraid it will not get noticed or add extra complexity otherwise. 

 For production environments, people can of course choose what to install.